devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -209,17 +222,138 @@ def ping() {
 }
 
 def refresh() {
-	delayBetween([
-		zwave.switchMultilevelV1.switchMultilevelGet().format(),
-		zwave.meterV2.meterGet(scale: 0).format(),
-		zwave.meterV2.meterGet(scale: 2).format(),
-	], 1000)
+    log.debug "refresh()"
+
+    encapSequence([
+        zwave.switchMultilevelV1.switchMultilevelGet(),
+        meterGet(scale: 0),
+        meterGet(scale: 2),
+    ], 1000)
 }
 
 def setLevel(level) {
 	if(level > 99) level = 99
-	delayBetween([
-		zwave.basicV1.basicSet(value: level).format(),
-		zwave.switchMultilevelV1.switchMultilevelGet().format()
+	encapSequence([
+		zwave.basicV1.basicSet(value: level),
+		zwave.switchMultilevelV1.switchMultilevelGet()
 	], 5000)
 }
+
+def configure() {
+	log.debug "configure()"
+    def result = []
+
+    log.debug "Configure zwaveInfo: "+zwaveInfo
+
+    if (zwaveInfo.mfr == "0086") {  // Aeon Labs meter
+    	// TEST CONFIGS
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 90, size: 1, scaledConfigurationValue: 1))) // enable threashhold values for wattage reporting
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 91, size: 2, scaledConfigurationValue: 0))) // min change in wattage
+		result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 92, size: 1, scaledConfigurationValue: 0))) // min change in wattage %
```

Comment:
Do these work for the older Aeon meters too? If not, this code should check the product id before sending them.

Reply:
We do not have older meters available for testing, hence, we will remove this and use the default values.